src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/CookieHelper.cs

```diff
@@ -22,7 +22,14 @@ public static void ProcessReceivedCookies(HttpResponseMessage response, CookieCo
                 Uri requestUri = response.RequestMessage.RequestUri;
                 for (int i = 0; i < valuesArray.Length; i++)
                 {
-                    cookieContainer.SetCookies(requestUri, valuesArray[i]);
+                    try
+                    {
+                        cookieContainer.SetCookies(requestUri, valuesArray[i]);
```

Comment:
Any way to call CookieCutter directly with the isThrow option disabled? https://referencesource.microsoft.com/#System/net/System/Net/_CookieModule.cs,83